global_vars.tf

```diff
@@ -22,10 +22,19 @@ variable "aws_alb_arn" {
   description = "The ARN of the ALB"
 }
 
+variable "alb_listener_path_pattern" {
+  description = "The path pattern to match to route to this service"
+  default = "/*"
+}
+
 variable "service_name" {
   description = "The service name used in the URL"
 }
 
+variable "slack_alert_sns_arn" {
+  description = "The ARN of sns topic for slack alerts"
```

Comment:
Should there be a default value for this? eq-terraform doesn't have this value configured yet so it'll error.

Reply:
No we cant default this really as it needs to be set.
if you look at branch `migrate-runner-to-ecs-deploy` on `eq-terraform` then it have it set in there. But that branch isn't ready for PR yet